### PR TITLE
[#181849579] Splitte Drainings in Infrastruktur / Applikation

### PR DESCRIPTION
**Pull request by eifelmicha**
- sleep um Consul Zeit für propagieren zu geben
- Applikationsdraining ebenfalls möglich
- Drainingtypen: `application` / `infrastructure`

### CODE REVIEW

**Comment by eifelmicha** [comment]
@dennis-kuypers 
requesting review.
requesting functional acceptance.
requesting merge approval.

File: tasks/main.yml [hunk 1]
```diff
@@ -50,7 +50,23 @@
...
+    - name: Create tomcat systemd drop-in graceful infrastructure shutdown
+      vars:
+        required_services:
+          - haproxy.service
+          - mongos.service
+            - docker.service
+            - consul-client.service
+            - dnsmasq.service
+            - systemd-resolved.service
+            - dbus.service # consul health checks use `systemctl` which requires `dbus` to be running
```
Comment (dennis-kuypers) [inline tasks/main.yml]: Indentation off?

**Review by dennis-kuypers — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (eifelmicha) [reply tasks/main.yml]: Ja

**Comment by eifelmicha** [comment]
@dennis-kuypers
requesting review.
requesting functional acceptance.
requesting merge approval.

**Review by dennis-kuypers — APPROVED** [review]
Review: LGTM
Functionally accepted
Merge approved

**Pull request merged**